.ci.yaml

```diff
@@ -281,6 +281,54 @@ targets:
       target_file: web_build_all_packages.yaml
       channel: stable
 
+  - name: Linux_web web_platform_tests_shard_1 master
+    bringup: true # New target
+    recipe: packages/packages
+    timeout: 60
+    properties:
+      add_recipes_cq: "true"
```

Comment:
I am wondering if we want to enforce these targets in the recipes CQ. To me, existing packages targets (with `add_recipes_cq: true`) should be enough to cover recipes change. I would suggest we skip this property here. This way, test specific regression will not block recipes CI.

Reply:
I can remove it.

For my future reference, what's the criteria for `add_recipes_cq`? Is it just a judgement call about whether the test depends on some aspect of the recipe that's not already covered?

Reply:
Yes, exactly. Since all these packages targets share the same recipes, and it's not necessary to enable all targets from recipes side.